### PR TITLE
Fix style errors

### PR DESCRIPTION
**Pull request by michelleN**

### CODE REVIEW

File: pkg/chart/chart.go [hunk 1]
```diff
@@ -426,18 +427,20 @@ func (c *Chart) loadMember(filename string) (*ChartMember, error) {
...
-type ChartContent struct {
-	Chartfile *Chartfile     `json:"chartfile"`
-	Members   []*ChartMember `json:"members"`
+//chartContent is abstraction for the contents of a chart
+type chartContent struct {
+	Chartfile *Chartfile `json:"chartfile"`
+	Members   []*Member  `json:"members"`
 }
 
-func (c *Chart) LoadContent() (*ChartContent, error) {
```
Comment (jackgr) [inline pkg/chart/chart.go]: Why make this private?

File: pkg/chart/chart.go [hunk 2]
```diff
@@ -426,18 +427,20 @@ func (c *Chart) loadMember(filename string) (*ChartMember, error) {
 	return result, nil
 }
 
-type ChartContent struct {
-	Chartfile *Chartfile     `json:"chartfile"`
-	Members   []*ChartMember `json:"members"`
+//chartContent is abstraction for the contents of a chart
```
Comment (jackgr) [inline pkg/chart/chart.go]: Why make this private?

File: pkg/chart/chart.go [hunk 3]
```diff
@@ -267,6 +267,7 @@ func Load(archive string) (*Chart, error) {
 	return LoadDataFromReader(raw)
 }
 
+// LoadDataFromReader loads a chart a reader
```
Comment (jackgr) [inline pkg/chart/chart.go]: from?

**Comment by jackgr** [comment]
@michelleN please see comments added just now.

File: pkg/chart/chart.go (hunk 1 above)
Comment (michelleN) [inline pkg/chart/chart.go]: It's not being used anywhere.

File: pkg/chart/chart.go (hunk 2 above)
Comment (michelleN) [inline pkg/chart/chart.go]: also not being used outside of the package. I use the practice "always private unless it has to be public" just to be safe.

File: pkg/chart/chart.go (hunk 3 above)
Comment (michelleN) [inline pkg/chart/chart.go]: yep I'll fix it.

File: pkg/chart/chart.go (hunk 1 above)
Comment (jackgr) [inline pkg/chart/chart.go]: But will be shortly by expansion.

File: pkg/chart/chart.go (hunk 2 above)
Comment (jackgr) [inline pkg/chart/chart.go]: See above.

File: pkg/chart/chart.go (hunk 1 above)
Comment (michelleN) [inline pkg/chart/chart.go]: Alrighty, you're welcome to change with your PR. :+1:

File: pkg/chart/chart.go (hunk 2 above)
Comment (michelleN) [inline pkg/chart/chart.go]: see above.

**Pull request merged**